### ISSUE TITLE
SyntaxError: Unexpected token '.'

### ISSUE THREAD

**Issue by Ndpnt** [superfaceai/release-changelog-action#249]
Since the version 1.0.2, I have this error:
```
Run superfaceai/release-changelog-action@v1
  with:
    path-to-changelog: CHANGELOG.md
    version: v1.1.1
    operation: release
  env:
    RELEASE_TYPE: patch
    NEW_VERSION: v1.1.1
    RELEASE_TAG: latest

/home/runner/work/_actions/superfaceai/release-changelog-action/v1/dist/index.js:3329
            return testComparator?.intersects(otherComparator, options);

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
    at internal/main/run_main_module.js:17:47    
```

**Comment by jnv** [comment]
Ah, nevermind, didn't realize the action specifies its environment.

@janhalama Here's your problem:

https://github.com/superfaceai/release-changelog-action/blob/b23716347531cf5755fcacbe70f682e7a3a9ffac/action.yml#L18

**Comment by janhalama** [comment]
The oldest supported LTS is 16 according to this changelog https://github.blog/changelog/2022-05-20-actions-can-now-run-in-a-node-js-16-runtime/

**Comment by janhalama** [comment]
@Ndpnt I fixed the error in version `v2.0.0`. New major version tag has been also released `v2`.

Please let me know if this solved it.

To reference new version of this Github Action bump version of it in your Github Actions script:
```
uses: @superfaceai/release-changelog-action@v2
```

**Comment by Ndpnt** [comment]
Yes it works, thanks to both of you for the responsiveness! 🙏
